recipe/meta.yaml

```diff
@@ -1,44 +1,41 @@
-{% set version = "0.21.1" %}
+{% set version = "0.22.0" %}
 
 package:
   name: cartopy
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/C/Cartopy/Cartopy-{{ version }}.tar.gz
-  sha256: 89d5649712c8582231c6e11825a04c85f6f0cee94dbb89e4db23eabca1cc250a
+  url: https://github.com/SciTools/cartopy/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7adf6b92f4e16c46fb1ce58840275af7a6dc1f21d5243d4b665ecef72961f9f4
 
 build:
   error_overlinking: true
-  skip: true  # [py<38]
-  number: 1
+  skip: true  # [py<39]
+  number: 0
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
-    - geos                                   # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - python
     - pip
+    - setuptools >=40.6.0
     - setuptools-scm >=7.0.0
-    - cython
+    - cython >=0.29.24
     - numpy
-    - geos
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - shapely >=1.6.4
+    - matplotlib-base >=3.4
+    - shapely >=1.7
+    - packaging >=20
     - pyshp >=2.1
-    - pyproj >=3.0.0
-    - geos
-    - matplotlib-base >=3.1
-    - scipy >=0.10
-    - ucrt  # [win]
```

Comment:
Scipy is only included as an extra in this version.  I'm not sure what `ucrt` was needed for but I didn't see any mention of it in this release.

Reply:
It is a universal C runtime and it was added in https://github.com/conda-forge/cartopy-feedstock/pull/116#issuecomment-912565452

The CIs are passing so I'm OK leaving it out. Hopefully we won't break cartopy for Windows users :-)

Reply:
@zklaus, you were the one to recommend adding this library.  It seemed to make CI pass at the time but doesn't seem to be needed now.  I didn't find any mention of it (outside of people listing their environments) upstream: https://github.com/search?q=repo%3ASciTools%2Fcartopy+ucrt&type=issues

Do you think it's okay to remove it at this point?

Reply:
IIRC, @zklaus is on holiday this month

Reply:
Ah, yes.  In that case maybe we assume the passing tests are a sign that this is no longer needed?  It seemed like the PR where this was added indicated that the Win tests only passed after it was added.

Reply:
I say merge and we can always add it back if someone reports it broken. Better to get this out there.

Reply:
I'll add that cartopy isn't doing anything special here on building (just a bit of Cython at this point), so if it is somehow needed, it really should be something else exporting the requirement.